src/main/java/com/getbase/sync/SyncService.java

```diff
@@ -49,6 +66,7 @@ public Session start(String deviceUUID) {
 
         // nothing new to synchronize
         if (response.getHttpStatus() == 204) {
```

Comment:
This condition might be extracted as method like `isSyncFinished()` not to duplicate it and for documentation purposes.